rosboard/handlers.py

```diff
@@ -57,6 +62,11 @@ def open(self):
             "hostname": socket.gethostname(),
             "version": __version__,
         }], separators=(',', ':')))
+        
+        # Publishers for active bot waypoint
+        qos =  rclpy.qos.QoSProfile(depth=1, durability=rclpy.qos.QoSDurabilityPolicy.TRANSIENT_LOCAL)
+        self.surveyor_waypoint_pub = rospy._node.create_publisher(Float32MultiArray, "/geosurvey/active_waypoint", qos)
+        self.digger_waypoint_pub = rospy._node.create_publisher(Float32MultiArray, "/digger/active_waypoint", qos)
 
```

Comment:
Looks good for now but later down we want to make sure we can handle multiple robots later

Reply:
Yeah, it also relies on there being bots of different species. Definitely needs to be revisited